docs/machineset-aws.md

```diff
@@ -1,9 +1,10 @@
 # Creating an AWS Windows MachineSet
 
-_\<windows_container_ami\>_ should be replaced with the AMI ID of a Windows
-image with the Docker container run-time installed. You must use Windows Server
-2019 with a version 10.0.17763.1457 or earlier to work around Windows containers
-behind a Kubernetes load balancer becoming unreachable
+_\<windows_container_ami\>_ can be replaced with the AMI ID of a Windows
+image with the Docker container run-time installed or without it as the default
+runtime will be containerd and installing containerd will be taken care by WMCO.
```

Comment:
In my opinion, if it doesnt matter if the image has or doesnt have docker its better just to drop mentioning docker at all